snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -303,3 +303,20 @@ instances:
         name: tcpPassiveOpens
         metric_tags:
           - TCP
+
+    ## @param metric_tags - list of elements - optional
+    ## Specify tags that you want applied to all metrics. It can be a symbol or a OID.
+    ##
+    ## - MIB: SNMPv2-MIB
+    ##   symbol: sysName
+    ##   name: snmp_host
+    ##
+    ## Or:
+    ##
+    ## - OID: 1.3.6.1.2.1.1.5
+    ##   name: snmp_host
+    #
+    # metric_tags:
+    #  - MIB: SNMPv2-MIB
+    #    symbol: sysName
+    #    name: snmp_host
```

Comment:
Does this show the default value for `metric_tags`? If not, maybe we can merge this with the examples above as…

```yaml
## @param metric_tags - list of elements - optional
## Specify tags that you want applied to all metrics. It can be a symbol or a OID.
##
## Example:
##
## metric_tags:
## - # From a symbol:
##   MIB: SNMPv2-MIB
##   symbol: sysName
##   name: snmp_host
## - # From an OID:
##   OID: 1.3.6.1.2.1.1.5
##   name: snmp_host 
```